mkosi/__init__.py

```diff
@@ -6733,6 +6932,32 @@ def generate_secure_boot_key(args: CommandLineArguments) -> NoReturn:
     os.execvp(cmd[0], cmd)
 
 
+def bump_image_version(args: CommandLineArguments) -> None:
+    """Write current image version plus one to mkosi.version"""
+
+    if args.image_version is None or args.image_version == "":
+        print("No version configured so far, starting with version 1.")
+        new_version = "1"
+    else:
+        v = args.image_version.split(".")
+
+        try:
+            m = int(v[-1])
+        except:
+            m = None
+
+        if m is None:
+            new_version = args.image_version + ".2"
+            print(
+                f"Last component of current version is not a decimal integer, appending '.2', bumping '{args.image_version}' → '{new_version}'."
+            )
+        else:
```

Comment:
Just because I noticed this, while scrolling through:
```suggestion
        try:
            m = int(v[-1])
        except ValueError:
            new_version = args.image_version + ".2"
            print(
                f"Last component of current version is not a decimal integer, appending '.2', bumping '{args.image_version}' → '{new_version}'."
            )
        else:
```
The `else` can be subsumed into the try-except. Python has `else` clauses on basically everything. For try-except it's "run this code iff no exception was raised"